http/server.ts

```diff
@@ -528,6 +528,18 @@ export class Server {
 
 /** Additional serve options. */
 export interface ServeInit {
+  /**
+   * Optionally specifies the address to listen on, in the form
+   * "host:port". The default is ":8000".
+   *
+   * If the port is omitted, `:80` is used by default for HTTP when invoking
+   * without `tls` options, and `:443` is
+   * used by default for HTTPS when invoking with `tls` options.
```

Comment:
Nit: `serveTls` is a separate function now.

Reply:
Good catch! Thanks